ui/de-lib/src/main/java/org/iplantc/de/apps/client/views/AppsViewImpl.java

```diff
@@ -25,39 +30,42 @@
     }
 
     @UiField(provided = true) final AppsToolbarView toolBar;
-    @UiField(provided = true) final AppCategoriesView appCategoriesView;
+    @UiField TabPanel categoryTabs;
```

Comment:
This is really the biggest change - I went from `AppsView` having the `AppCategoriesView` directly to having just a TabPanel.  The TabPanel then gets passed to `AppCategoriesPresenter` and `OntologyHierarchiesPresenter` to add whatever tabs they need to add.